ext/RMagick/extconf.rb

```diff
@@ -3,7 +3,12 @@
 
 module RMagick
   class Extconf
-    require 'rmagick/version'
+    if RUBY_VERSION < '1.9.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.